notebooks/tutorials/EventDisplayInteractive.ipynb

```diff
@@ -691,6 +691,9 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
    "version": "3.8.16"
+  },
+  "nbsphinx": {
+    "execute": "never"
   }
```

Comment:
Here
```
"nbsphinx": {
    "execute": "never"
  }
```

is to disable the execution of the notebook when readthedocs building.